core/ErrorQueue.h

```diff
@@ -26,7 +28,6 @@ class ErrorQueue {
     spdlog::logger &tracer;
     std::atomic<bool> hadCritical{false};
     std::atomic<int> nonSilencedErrorCount{0};
-    std::atomic<int> filesFlushedCount{0};
```

Comment:
This is another change that would have been good to land as a separate change, to showcase that there are no users of this outside of an ENFORCE that will soon need to be deleted.